projects/dev.yaml

```diff
@@ -0,0 +1,85 @@
+# RESTBase project config for development environments
+paths:
+  /{api:v1}:
+    x-modules:
+      - spec:
+          info:
+            version: 1.0.0
+            title: Wikimedia REST API
+            description: Welcome to your RESTBase API.
+          x-route-filters:
+            - path: ./lib/normalize_title_filter.js
+              options:
+                redirect_cache_control: '{{options.purged_cache_control}}'
+          paths:
+            /page:
+              x-modules:
+                - path: v1/content.yaml
+                  options: '{{options}}'
+                - path: v1/content_segments.yaml
+                  options:
+                      purged_cache_control: '{{options.purged_cache_control}}'
+                      cx_host: '{{options.transform.cx_host}}'
+                - path: v1/common_schemas.yaml # Doesn't really matter where to mount it.
+                - path: v1/mobileapps.yaml
+                  options: '{{merge({"response_cache_control": options.purged_cache_control},
+                                options.mobileapps)}}'
+            /transform:
+              x-modules:
+                - path: v1/transform.yaml
+                - path: v1/transform-lang.js
+                  options: '{{ merge({"cx_host": "http://localhost:12345/"}, options.transform) }}'
```

Comment:
Ulalala... Why are we merging it with some random port? If it is supported in dev env, the proper config should be provided. If it's not - the whole section regarding the cxserver must not be here.